src/weather.cpp

```diff
@@ -921,29 +923,24 @@ std::string get_wind_desc( double windpower )
 
 rl_vec2d convert_wind_to_coord( const int angle )
 {
-
-    rl_vec2d windvec;
-    float fx = sin( angle * M_PI / 180.0f );
-    float fy = -cos( angle * M_PI / 180.0f );
-    int roundedx;
-    int roundedy;
-    if( fx > 0.5 ) {
-        roundedx = 1;
-    } else if( fx < -0.5 ) {
-        roundedx = -1;
-    } else {
-        roundedx = 0;
-    }
-    if( fy > 0.5 ) {
-        roundedy = 1;
-    } else if( fy < -0.5 ) {
-        roundedy = -1;
-    } else {
-        roundedy = 0;
+    const std::array<std::pair<int, rl_vec2d>, 9> outputs = {{
+            { 330, rl_vec2d( 0, -1 ) },
+            { 301, rl_vec2d( -1, -1 ) },
+            { 240, rl_vec2d( -1, 0 ) },
+            { 211, rl_vec2d( -1, 1 ) },
+            { 150, rl_vec2d( 0, 1 ) },
+            { 121, rl_vec2d( 1, 1 ) },
+            { 60, rl_vec2d( 1, 0 ) },
+            { 31, rl_vec2d( 1, 1 ) },
+            { 0, rl_vec2d( 0, -1 ) }
```

Comment:
Numbers are wrong
![image](https://user-images.githubusercontent.com/1931904/60264341-79786c00-98d2-11e9-973e-24b63f1f4c62.png)

Reply:
Numbers are accurate to previous implementation. I'm aware of the heavy bias towards cardinal directions with the 60 deg cardinal arcs and 30 deg diagonal arcs.

Reply:
I could reduce the bias pretty easily, but doing so is outside the scope of this PR.
Cardinal arcs would become 46 deg, with 23 deg half arcs
Diagonal arcs would become 44 deg, with 22 deg half arcs
These values are close enough to correct, and don't require changing from integer to floating point representation.

Reply:
Okay. Than only one thing: it should be `{ 31, rl_vec2d( 1, -1 ) }`.

Reply:
ah, thanks for catching that!